### DISCUSSION TITLE
SD card log groups and variables

### DISCUSSION THREAD

**Discussion by anoopk18** [bitcraze/discussions#1591] [Q&A]
Hi,

I am logging groups and variables using sd card and I have vicon setup for localization. Firstly, I would like to thank all previous discussions regarding sd logging, all those have been extremely helpful for my setup. 
1) I referred to this [documentation](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/): and I wanted to clarify the difference between stateEstimate group and vicon data. In other words, I am trying to understand where the stateEstimate group variable gets data from and how it differs from vicon data. 
2) based on the same documentation above, I was trying to understand the difference between stateEstimate and stabilizer groups. The description says that it stores the same data as stateEstimate. 
3) how does stabilizer.thrust compare to motor.m1 - m4 groups? Does stabilizer.thrust equate to thrust from the entire crazyflie in Gs
4) Lastly, is there a timestamp attached to each of these groups and the data that is being stored? I didn't see one described in the config file. 

Thanks!

**Comment by gemenerik** [comment]
Regarding 3, check out https://github.com/orgs/bitcraze/discussions/1528

**Comment by gemenerik** [comment]
2. I think there may be some overlap between logs from the stateEstimate and stabilizer groups. It's probably easiest to verify this by finding the relevant log parameters [in the firmware.](https://github.com/bitcraze/crazyflie-firmware)
